docs/io/fits/appendix/faq.rst

```diff
@@ -475,6 +475,15 @@ by calling :func:`gc.collect` at the end of the loop.
 In a future release it will be more convenient to automatically perform this
 sort of cleanup when closing FITS files, where needed.
 
+Using header['NAXIS2'] += 1 does not add another row to my Table
+----------------------------------------------------------------
+
+`NAXIS` and similar keywords are FITS *structural* keywords and should not be
+modified by the user. They are automatically updated by :mod:`astropy.io.fits`
+when checking the validity of the data and headers. See `this discussion of
+structural keywords <../index.html#structural-keywords>`__ for more information.
```

Comment:
Please use the `:ref:` directive for linking here.

Reply:
Ah, that's perfect. Thanks!